src/Constants.ts

```diff
@@ -150,6 +150,17 @@ export class Constants {
     },
   };
 
+  public static panels = {
+    log: {
+      viewType: 'truffle-vscode.panel.log',
+      tool: {
+        truffle: 'truffle',
+        ganache: 'ganache',
+        dashboard: 'dashboard',
```

Comment:
- Would it be possible to move this into `LogView`, since it's only used there?
- Moreover, could we make it an `enum`?  since it's used that way.